content/docs/pipelines/tutorials/cloud-tutorials.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Run a Cloud-specific Pipelines Tutorial"
+description = "Choose the Kubeflow Pipelines tutorial to suit your deployment"
+weight = 1
++++
+
+{{% alert title="Opportunity to add cloud tutorials" color="info" %}}
+We currently have only a GCP tutorial for Kubeflow Pipelines. A tutorial for
+Microsoft Azure is on its way (see 
+<a href="https://github.com/kubeflow/website/pull/956">PR #956</a>). 
+<b>Invitation:</b> Create a cloud-specific tutorial and link it here.
```

Comment:
Nit: the "Invitation" doesn't render on a new line, but would look a little better if it did.

Reply:
This would also be a good place to link to the contributor's guide or a tutorial template. Maybe at some point there will be tutorial templates for each major cloud provider?

Reply:
Thanks! Both done.